js-code-guidelines.md

```diff
@@ -129,6 +130,12 @@ Always run tests before pushing and PR'ing your code.
 
 ### Guidelines
 
+#### Supported Versions
+
+IPFS projects support both the Current and Active LTS versions of Node.js.  Please see [nodejs.org](https://nodejs.org/) for what these currently are.
```

Comment:
Better link for the release schedule: https://github.com/nodejs/Release#release-schedule